datasets/fr/tresor/fr_tresor_gels_avoir.yml

```
@@ -4182,3 +4182,12 @@ lookups:
         props:
           jurisdiction: ir
           incorporationDate: 2014
+      - match: "Type d’entité: gang haïtien opérant principalement dans les départements de l’Artibonite et du Nord-Ouest: /"
+        props:
+          legalForm: gang haïtien
```

```suggestion
          description: gang haïtien
```

I wouldn't say a Haitian gang is a legislated form of organisaion, other than being banned :)
You could put gang haïtien in description, perhaps?